test-data/unit/check-newtype.test

```diff
@@ -295,15 +295,16 @@ a = Foo(type(3))
 [out]
 
 [case testNewTypeWithTypeVarsFails]
+# flags: --new-semantic-analyzer
 from typing import NewType, TypeVar, List
 T = TypeVar('T')
 A = NewType('A', T)
 B = NewType('B', List[T])
 [builtins fixtures/list.pyi]
 [out]
-main:3: error: Argument 2 to NewType(...) must be subclassable (got T?)
-main:3: error: Invalid type "__main__.T"
-main:4: error: Invalid type "__main__.T"
+main:4: error: Argument 2 to NewType(...) must be subclassable (got T?)
+main:4: error: Invalid type "__main__.T": Can only use bound type variables as types
+main:5: error: Invalid type "__main__.T": Can only use bound type variables as types
```

Comment:
Bikeshedding: `Type variable "__main__.T" is unbound`. Maybe also add a note about what this means, since unbound/bound can be confusing for new users. Idea:

```
main:4: error: Type variable "__main__.T" is unbound
main:4:note: (Hint: Use "Generic[T]" or "Protocol[T]" base class to bind "T" inside a class)
main:4:note: (Hint: Use "T" in function signature to bind "T" inside a function)
```